workflows/sarek.nf

```diff
@@ -60,6 +60,10 @@ if (params.wes && !params.step == 'annotate') {
     else log.warn("Intervals file was provided without parameter `--wes`: Pipeline will assume this is Whole-Genome-Sequencing data.")
 } else if (params.intervals && !params.intervals.endsWith("bed") && !params.intervals.endsWith("interval_list")) exit 1, "Intervals file must end with .bed or .interval_list"
 
+if(params.aligner && params.aligner.contains("dragmap") && !(params.skip_tools && params.skip_tools.contains("baserecalibrator"))){
+    log.warn("DragMap was specified as aligner. Base recalibration is not contained in `--skip_tools`. It is recommended to skip baserecalibration when using DragMap\nhttps://gatk.broadinstitute.org/hc/en-us/articles/4407897446939--How-to-Run-germline-single-sample-short-variant-discovery-in-DRAGEN-mode")
```

Comment:
shouldn't we have an error there instead of a warning?
I'd go for `step == "mapping` also.
Also, no need to check if `params.aligner` exists, as default is `bwa`

Reply:
I wouldn't want to add an error. Maybe someone has a special edge case where they don't follow the guidelines. I guess it is not forbidden to do recalibration

Reply:
good point